app.py

```diff
@@ -107,6 +107,42 @@ def add_ekubo_liquidity(
     return data
 
 
+def create_stablecoin_bundle(data: pandas.DataFrame):
+    for collateral in src.settings.COLLATERAL_TOKENS:
+        relevant_pairs = [pair for pair in data.keys() if collateral in pair and any(stablecoin in pair for stablecoin in src.settings.DEBT_TOKENS[:-1])]
+        combined_df = None
+
+        for pair in relevant_pairs:
+            df = data[pair]
+
+            if 'collateral_token_price' not in df.columns:
+                logging.warning(f"'collateral_token_price' is missing in DataFrame for pair: {pair}")
+                break
+                
+            if combined_df is None:
+                combined_df = df.copy()
+
+            else:
+                if 'collateral_token_price' not in combined_df.columns:
+                    logging.warning("'collateral_token_price' is missing in the combined DataFrame before merging.")  
+                    break
+
+                combined_df = pandas.merge(combined_df, df, on='collateral_token_price', suffixes=('', '_y'))
+
+                for col in ['liquidable_debt', 'liquidable_debt_at_interval', 
+                            '10kSwap_debt_token_supply', 'MySwap_debt_token_supply', 
+                            'SithSwap_debt_token_supply', 'JediSwap_debt_token_supply', 
+                            'debt_token_supply']:
+                    combined_df[col] += combined_df[f'{col}_y']
+
+                combined_df.drop([col for col in combined_df.columns if col.endswith('_y')], axis=1, inplace=True)
+
+        new_pair = f'{collateral}-(All USD Stable Coins)'
```

Comment:
Please define global variable for bundle name at the top of this module
```python
STABLECOIN_BUNDLE_NAME = '(All USD Stable Coins)'
```
and use it everywhere instead of string. (purpose is that if we decide to rename bundle in future, we just change the name at single place)

Reply:
Done! ✅